### ISSUE TITLE
IOS app?

### ISSUE THREAD

**Issue by ebdavison** [gjedeer/Acastus#25]
I know this is an Android app and that it is propably pretty specific code-wise for that platform but wanted ask if there has been any thought to making this into and IOS app as well.  I cannot find a similar app for IOS and I have heard good things about this.

Maybe a flutter app would do the trick?  I have heard that flutter can work on both IOS and Android ... Not a flutter nor mobile developer myself or I would take a stab at it.

**Comment by gjedeer** [comment]
I know literally one person who owns an iphone. The only time I actually held an iDevice was around 10 years ago playing Fruit Ninja on a friend's ipad. So i don't have any experience programming any devices, or incentive to cater to that niche.

It shouldn't be too hard though, this app is just formatting JSON replies in a list box, may be a good learning project for you!

**Comment by ebdavison** [comment]
Yes, that very well might be the case and I understand your point.  I am quite familiar with JSON so the point is probably a good one about a learning project?

What part of the response from pelias are you passing through to OsmAnd ?  The lat/long?
